src/commons_codec/transform/dynamodb.py

```diff
@@ -170,24 +173,23 @@ def update_clause(self, record: t.Dict[str, t.Any]) -> SQLParameterizedClause:
         data['humidity'] = '84.84', data['temperature'] = '55.66'
         """
 
-        clause = SQLParameterizedClause()
+        clause = SQLParameterizedSetClause()
         for column, value in record.items():
             param_name = f":{column}"
+            rval = None
             if value is None:
                 value = "NULL"
```

Comment:
Despite its intentions, this is probably not correct. Better remove that?

Reply:
This has been removed. `None`-values on the input side are no longer treated specially.